releases/unreleased/Use_entry_points_to_support_plugin_mechanism.yml

```
@@ -0,0 +1,5 @@
+title: 'Use entry points to support plugin mechanism'
+category: New feature
+author: ChenqiShan <chenqishan337@gmail.com>, LinHaiming <lhming23@outlook.com>
+issue: 
+notes: The plugin solution is used to incubate a new data source backend solution. 
```

```suggestion
notes: >
    This plugin mechanism allows ELK to use third-party enriching backends.
    Developers only need to include the next snippet in their package configuration
    and ELK will import them as any other core backend. 
    ```
    entry_points={"grimoire_elk": "mybackend = package.subpackage:get_connectors"},
    ```
    **NOTE**: In the previous example, `get_connectors` is a function that returns
    the backend classes needed to enrich a datasource. Take the next snipped
    as an example:
    ```
    def get_connectors():
        return {"backend": [MyBackend, MyBackendOcean, MyBackendEnrich, MyBackendCommand]}
    ```
```